tests/test_digest_handler.py

```
@@ -42,290 +65,368 @@ def _make_vault(tmp_path: Path) -> Path:
-
-    ctx = TaskContext(
-        vault_dir=vault, task_path=task, prefix="DIGEST",
-        slug="daily", body="", pack="research-tech", llm_client=llm,
-    )
-    handle_digest(ctx)
-
+    as_of = datetime.now(timezone.utc) - timedelta(minutes=30)
+    _seed_window_with_signal(vault, as_of=as_of)
+    llm = _FakeLLM("body")
+    handle_digest(_ctx(vault, llm))
     sys_prompt, user_prompt = llm.calls[0]
```

Ruff (0.15.12)

[warning] Unpacked variable `sys_prompt` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)